src/openrct2-android/app/build.gradle

```diff
@@ -45,28 +45,25 @@ android {
         }
     }
 
+    flavorDimensions "version"
+    productFlavors {
+        full {
+            dimension "version"
+        }
+    }
+
     externalNativeBuild {
         cmake {
             path 'src/main/CMakeLists.txt'
         }
     }
 
     productFlavors {
-        arm7 {
-            ndk {
-                abiFilters 'armeabi-v7a'
-            }
-        }
         arm {
             ndk {
                 abiFilters 'armeabi-v7a', 'arm64-v8a'
             }
         }
-        x86 {
```

Comment:
I take it you wish to focus on building for ARM only?

Given the Intel market share for Android tablets, or lack thereof, I don't see a problem with the change; I'd just like to make sure.

Reply:
oh, that was not meant to be part of PR

Reply:
ChromeOS is largely x86/x86-64 and they can run Android applications

Reply:
Ah, right! Well, glad we're doing code review, then! :+1: